src/python/pants/backend/docker/goals/package_image.py

```diff
@@ -65,7 +64,7 @@ class DockerImageOptionValueError(ValueError):
 
 
 @dataclass(frozen=True)
-class DockerFieldSet(PackageFieldSet, RunFieldSet):
+class DockerFieldSet(PackageFieldSet):
```

Comment:
This would probably benefit from a rename to `DockerPackageFieldSet`

Reply:
Yep, good call.